edge-util/test/Microsoft.Azure.Devices.Edge.Util.Test.Common/settings/base.json

```diff
@@ -13,11 +13,11 @@
   "testSuite": [
     "test-config-x64.json"
   ],
-  "dockerHostUrl": "http://localhost:2375",
+  "dockerHostUrl": "unix://var/run/docker.sock",
   "GatewayHostname": "127.0.0.1",
   "GatewayDeviceId": "device1",
   "StressTest_MessagesCount_SingleSender": "10000",
   "StressTest_MessagesCount_MultipleSenders": "5000",
   "StressTest_MessagesCount_MultipleSendersMultipleReceivers": "5000",
   "Tests_StartEdgeHubService": true
-}
+}
```

Comment:
@damonbarry FYA I updated the client id and thumbprint to make it so that we can run CI tests locally without interacting with these files. IIUC windows doesn't even work so I can go ahead and remove the settings/windows.json.